go-controller/pkg/cluster/node.go

```diff
@@ -98,7 +100,7 @@ func (cluster *OvnClusterController) StartClusterNode(name string) error {
 // IP address
 func (cluster *OvnClusterController) updateOvnNode(masterIP string,
```

Comment:
@JacobTanenbaum I don't think cluster.updateOvnNode() is used anymore? Maybe we can just remove the function.

Reply:
Looking at it more, it looks like updateOvnNode() assumes that the master OVN database will be empty and that it should re-create its management port and gateway stuff. I think we should check with @girishmg and @shettyg to see whether their HA setup will do that.

Reply:
@dcbw That code was added to handle new master nodes with new IP.  It was one of the HA strategies - but is not complete and has holes in it. We can consider removing it, if maintaining it is painful.

Reply:
@shettyg Girish, I and others on a recent community meeting decided to remove the existing HA strategy since neither of us (nvidia, RH) were using it. That code in the node would go away anyway when all logical network creation happens in the ovnkube master too, and then the master would be responsible for recreating the logical network in an empty DB during reconciliation.

Does removal now and then doing (a) move logical net stuff to master and (b) reconciliation in master sound OK to you? We need to do both these things anyway.

Reply:
I am OK with that.